docs/data-management.rst

```diff
@@ -280,38 +426,78 @@ Step-by-Step
 
-#. The variables ``$ID​1`` and ``$ID2`` below are the IDs of the ​selected Datasets. To duplicate two Datasets with their Images and annotations on both the Images and the Datasets, run::
+#. The variables ``$ID​1`` and ``$ID2`` below are the 
+   IDs of the ​selected Datasets. To duplicate two Datasets
+   with their Images and annotations on both the 
+   Images and the Datasets, run::
     
    $ ​omero duplicate Dataset:$ID1,$ID2 --report
 
-#. The duplicated Datasets will not be linked to any Project, even if the originals were linked to some Project. 
+#. The duplicated Datasets will not be linked to any Project,
+   even if the originals were linked to some Project. 
 
-#. Duplicate two Images with many ROIs on them. The ROIs duplication might take a long time. To exclude the duplication of the ROIs, run::
+#. Duplicate two Images with many ROIs on them. 
+   The ROIs duplication might take a long time.
+   To exclude the duplication of the ROIs, run::
 
    $ omero duplicate Image:$ID1,$ID2 --ignore Roi --report
 
-#. Find the duplicated Images in the Orphaned Images and Drag and Drop them into a Dataset or create a new Dataset for them.
-
-#. Duplicate two Projects of another user in read-annotate group type. The group name in our example below is `read-annotate-group`. Log in as `user-1` and duplicate Projects belonging to `user-2` in that group. Further, instead of duplicating all the annotations of `user-2`, link all the original annotations such as Tags, Key-Value pairs and FileAnnotations of `user-2` to the corresponding objects in the duplicate, and duplicate only Comments and Ratings. The duplicate and Comments and Ratings on it will belong to `user-1`, but the other annotations will still belong to `user-2`. Run::
+#. Find the duplicated Images in the Orphaned Images
+   and Drag and Drop them into a Dataset or create a new Dataset for them.
+
+#. Duplicate two Projects of another user in read-annotate group type.
+   This will duplicate also the Datasets linked to those Projects
+   as welll as Images linked to those Datasets. 
+   Here we specify some reference classes of objects that
+   we do not want to duplicate, even though they are linked to the objects
+   we are duplicating. Instead, the duplicated objects
+   will be linked to these `reference-classes` of objects. 
+   We can also specify classes that we explicitly do want to duplicate
+   using the `duplicate` argument. In this example,
+   we specify all Annotations as reference-classes,
+   but a subset of these (Comments and LongAnnotations such as Ratings)
+   are `duplicate-classes`. This means that the duplicated Projects
+   will be linked to the original annotations such as Tags,
+   Key-Value pairs and FileAnnotations, but Comments and Ratings
+   will be duplicated. This can be useful to ensure that a single
+   Comment is not attached to multiple objects which might cause confusion
+   when the comment is edited. The duplicated Annotations
```

Comment:
and the link to the comment might be severed if the image is moved to another group!

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/17/commits/27c933f08240fcf6e8c760a3f5956cca51d594b4